core/src/main/scala/fs2/interop/reactivestreams/StreamSubscriber.scala

```diff
@@ -16,8 +16,8 @@ import scala.concurrent.ExecutionContext
   *
   * @see https://github.com/reactive-streams/reactive-streams-jvm#2-subscriber-code
   */
-final class StreamSubscriber[F[_], A](val sub: StreamSubscriber.FSM[F, A])(implicit A: Effect[F],
-                                                                           ec: ExecutionContext)
+final class StreamSubscriber[F[_], A](val sub: StreamSubscriber.FSM[F, A])(implicit F: ConcurrentEffect[F],
+                                                                           timer: Timer[F])
```

Comment:
I was a bit confused as to why `ConcurrentEffect` was here when we're not using any of its methods. I see now that `sub` requires a `Concurrent` instance which is supplied on creation, so I suppose the subscriber could explicitly require a `ConcurrentEffect` to indicate concurrency.